demo/data.ts

```diff
@@ -1,105 +1,148 @@
-export let single = [
+export const single = [
   {
     name: 'Germany',
-    value: 46268
+    region: 'Europe and Central Asia',
+    value: 40632,
+    lifeExpectancy: 80.3,
+    population: 80.4
   },
   {
     name: 'USA',
-    value: 53041
+    region: 'America',
+    value: 49737,
+    lifeExpectancy: 78.8,
+    population: 310
   },
   {
     name: 'France',
-    value: 42503
+    region: 'Europe and Central Asia',
+    value: 36745,
+    lifeExpectancy: 81.4,
+    population: 63
   },
   {
     name: 'United Kingdom',
-    value: 41787
+    region: 'Europe and Central Asia',
+    value: 36240,
+    lifeExpectancy: 80.2,
+    population: 62.7
   },
   {
     name: 'Spain',
-    value: 29863
+    region: 'Europe and Central Asia',
+    value: 33000,
+    lifeExpectancy: 81.8,
+    population: 46.6
   },
   {
     name: 'Italy',
-    value: 35925
+    region: 'Europe and Central Asia',
+    value: 35800,
+    lifeExpectancy: 81.9,
+    population: 59.6
   }
 ];
 
-export let multi = [
+export const multi = [
```

Comment:
Using the `multi` dataset for the bubble chart.  The demos should show the properly formatted input.